README.md

```diff
@@ -1,30 +1,130 @@
+            'type': 'string'
+        }, 
+        {
+            'name': 'trades', 
+            'type': {
+                'type': 'array', 
+                'items': ['string', 'int']
+            }
+        }
+    ]
+}
+```
+
+CORRESPONDING CODE
+```
+def serialize(data, output):
+    cdef bytearray buffer = bytearray()
+    cdef dict datum
+    cdef str type_0
+    write.write_string(buffer, data['order_id'])
+    if len(data['trades']) > 0:
+        write.write_long(buffer, len(data['trades']))
+        for val_0 in data['trades']:
+            if type(val_0) is tuple:
+                type_0, val_1 = val_0
+                
+                if type_0 == 'string':
+                    write.write_long(buffer, 0)
+                    write.write_string(buffer, val_1)
+                
+                elif type_0 == 'int':
+                    write.write_long(buffer, 1)
+                    write.write_int(buffer, val_1)
+                
+            else:
+                if type(val_0) is str:
+                    write.write_long(buffer, 0)
+                    write.write_string(buffer, val_0)
+                elif type(val_0) is int:
+                    write.write_long(buffer, 1)
+                    write.write_int(buffer, val_0)
+    write.write_long(buffer, 0)
+    output.write(buffer)
+
+
+
+def deserialize(fo):
+    cdef long long i_0
+    cdef long long i_1
+    cdef long i_2
+    data = {}
+    data['order_id'] = read.read_string(fo)
+    data['trades'] = []
+
+    i_1 = read.read_long(fo)
+    while i_1 != 0:
+        if i_1 < 0:
+            i_1 = -i_1
+            read.read_long(fo)
+        for i_0 in range(i_1):
+            i_2 = read.read_int(fo)
+            if i_2 == 0:
+                val_2 = read.read_string(fo)
+            if i_2 == 1:
+                val_2 = read.read_int(fo)
+            data['trades'].append(val_2)
+        i_1 = read.read_long(fo)
+    return data
+```
+
+
 **Usage Example:**
-1. Put all your schemata into a schemata root folder following this pattern.<br>
-`[schemata_root_folder]/[namespace]/[schema_name]/[schema_version]/schema.yaml`<br>
-It is essential that you follow this pattern and safe your schemata in `schema.yaml` files.
-
-2. Run `cerializer_handler.update_cerializer` with a list of all your schemata root directories.<br>
-eg. `cerializer_handler.update_cerializer(['cerializer/tests/schemata'])`<br>
-This will generate an importable module for each schema that contains two methods: `serialize` and `deserialize`.<br>
-This module can be then imported as:<br>
- `import [schema_name]_[schema_version]`<br>
- and then used accordingly<br>
-3. Use the module accordingly.<br>
-  eg.<pre><code>
- import student_schema_1
- import io<br>
- output = io.BytesIO()
- data_recod = {
+1. Create an instance of CerializerSchemata
+For initializing CerializerSchemata it is necessary to supply a list of tuples in form of (schema_identifier, schema)
+schema touple = (namespace.schema_name, {schema})
```

Comment:
Typo touple -> tuple

Reply:
You could describe what `{schema}` is. As it is written it seems that it is a set of one element `schema` in it. You could write down that the schema is a python dict containing the avro schema.